sdc/tests/test_series.py

```diff
@@ -3495,6 +3495,94 @@ def test_impl(S1, S2, S3):
         pd.testing.assert_series_equal(hpat_func(S1, S2, S3),
                                        test_impl(S1, S2, S3))
 
+    # sdc.datatypes.common_functions.sdc_join_series_indexes work uncorrectly
+    # different indexes
+    @unittest.expectedFailure
+    def sdc_join_series_indexes(self):
+        def test_impl(S1, S2):
+            return S1.add(S2)
+
+        sdc_func = self.jit(test_impl)
+
+        data = [0, 1, 2, 3, 4]
+        index1 = [3.3, 5.4, np.nan, 7.9, np.nan]
+        index2 = [3, 4, 3, 9, 2]
+        S1 = pd.Series(data, index1)
+        S2 = pd.Series(data, index2)
+        pd.testing.assert_series_equal(sdc_func(S1, S2), test_impl(S1, S2))
+
+    @unittest.skip('SDC returns only float Series')
```

Comment:
Generally it's no good to add some functionality and keep tests for it skipped, so the same comment as above: used check_dtype=False.